doppelganger/doppelganger_state.go

```diff
@@ -0,0 +1,28 @@
+package doppelganger
+
+// doppelgangerState tracks the validator's state in Doppelganger Protection.
+type doppelgangerState struct {
```

Comment:
Like I commented in `doppleganger.go`, I think this should just be `state` , also the filename.

Reply:
https://github.com/ssvlabs/ssv/pull/2036#discussion_r1969601922